using-timescaledb/continuous-aggregates.md

```diff
@@ -248,6 +244,111 @@ hypertable with a continuous aggregate.
+millifortnight (which is about 20 minutes).
+
+```sql
+CREATE TABLE devices(
+  time BIGINT,        -- Timestamp in microfortnights
+  cpu_usage INTEGER,  -- Total CPU usage
+  disk_usage INTEGER, -- Total disk usage
+  PRIMARY KEY (time)
+);
+
+SELECT create_hypertable('devices', 'time',
+                         chunk_time_interval => 1000);
+```
+
+To define a continuous aggregate on a hypertable that is using an integer
+dimension, it is necessary to have a function to get the current
+timestamp in whatever representation that you are using and set it for
+the hypertable using
+[`set_integer_now_func`][api-set-integer-now-func]. The function can
+be defined as a normal PostgreSQL function, but needs to: be
+[`STABLE`][pg-func-stable], take no arguments, and return an integer
+of the same type as the time column in the table. In our case, this
+should suffice:
+
+```sql
+CREATE OR REPLACE FUNCTION current_microfortnight() RETURNS BIGINT
+LANGUAGE SQL STABLE AS $$
+	SELECT CAST(1209600 * EXTRACT(EPOCH FROM CURRENT_TIMESTAMP) / 1000000 AS BIGINT)
+$$;
+
+SELECT set_integer_now_func('devices', 'current_microfortnight');
+```
+
+Once the replacement for current timestamp has been set up, you can
+define the continuous aggregate for the `devices` table.
+
+```sql
+CREATE VIEW devices_summary
+WITH (timescaledb.continuous) AS
+SELECT time_bucket('500', time) AS bucket,
+       avg(cpu_usage) AS avg_cpu,
+       avg(disk_usage) AS avg_disk
+     FROM devices
+     GROUP BY bucket;
+```
+
+You can now insert some rows to check if the aggregation works as
+expected.
+
+```sql
+CREATE EXTENSION tablefunc;
+
+INSERT INTO devices(time, cpu_usage, disk_usage)
+SELECT time,
+       normal_rand(1,70,10) AS cpu_usage,
+	   normal_rand(1,2,1) * (row_number() over()) AS disk_usage
+  FROM generate_series(1,10000) AS time;
+```
+
+>:TIP: You can use the `tablefunc` extension to generate a Gaussian
+>distribution and use the `row_number` function to turn it into a
+>cumulative sequence.
+
+You can now check that the view contains the correct data. 
+
+```sql
+postgres=# SELECT * FROM devices_summary ORDER BY bucket LIMIT 10;
+ bucket |       avg_cpu       |       avg_disk
+--------+---------------------+----------------------
+      0 | 63.0000000000000000 |   6.0000000000000000
+      5 | 69.8000000000000000 |   9.6000000000000000
+     10 | 70.8000000000000000 |  24.0000000000000000
+     15 | 75.8000000000000000 |  37.6000000000000000
+     20 | 71.6000000000000000 |  26.8000000000000000
+     25 | 67.6000000000000000 |  56.0000000000000000
+     30 | 68.8000000000000000 |  90.2000000000000000
+     35 | 71.6000000000000000 |  88.8000000000000000
+     40 | 66.4000000000000000 |  81.2000000000000000
+     45 | 68.2000000000000000 | 106.0000000000000000
+(10 rows)
+```
+
+Note that the continuous aggregate will lag behind the hypertable 500
+microfortnights (since the bucket size is 500 microfortnights) so you
+have to wait for it to be updated.
```

Comment:
Why? It is the doc for 1.7, where real time aggregates are added. Should the query produce the complete result?